### PR TITLE
Add check to address 8210 review

### PR DESCRIPTION
**Pull request by paulb777**
#no-changelog

### CODE REVIEW

File: FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m [hunk 1]
```diff
@@ -263,26 +263,28 @@ - (nullable NSString *)reCAPTCHATokenForURL:(NSURL *)URL error:(NSError **_Nonnu
...
-                                               message:errorDict[@"message"]];
-  if (!*error) {
-    NSString *reason;
-    if (errorDict[@"code"] && errorDict[@"message"]) {
-      reason = [NSString stringWithFormat:@"[%@] - %@", errorDict[@"code"], errorDict[@"message"]];
-    } else {
-      reason = [NSString stringWithFormat:@"An unknown error occurred with the following "
-                                           "response: %@",
-                                          deepLinkURL];
+  if (error != nil) {
```
Comment (maksymmalyhin) [inline FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m:266]: nit: technically not a bug, but usually pointers are compared with `NULL`.

**Review by maksymmalyhin — COMMENTED** [review]
LGTM 👍  Thank you for addressing it!

**Review by maksymmalyhin — APPROVED** [review]

File: FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m (hunk 1 above)
Comment (maksymmalyhin) [reply FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m]: thanks!

Comment (paulb777) [reply FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m]: Good catch! Fixed.

**Pull request merged**